packages/rocketchat-apps/client/admin/appManage.js

```diff
@@ -5,6 +5,16 @@ import { AppEvents } from '../communication';
 
 
 Template.appManage.onCreated(function() {
+
+	if (RocketChat.settings.cachedCollectionPrivate == null) {
```

Comment:
What's this for in this file? As I don't see it being used anywhere except the commented out code below on line 87..